generator/src/main/java/com/scottlogic/deg/generator/generation/VelocityMonitor.java

```diff
@@ -87,7 +85,7 @@ public void endGeneration() {
             averageRowsPerSecond);
 
         println(
-            "\nGeneration finished at: %s",
+            "\nGeneration finished at: %s UTC",
```

Comment:
Slight issue here: you have changed it to print out local time, but you've labelled it as UTC.  You should take out this change (and optionally print out the correct abbreviation for the timezone that the `ZonedDateTime` object is using.